### ISSUE TITLE
[Bug]: iOS - Swap transaction failed due Get XOR on SORA Card

### ISSUE THREAD

**Issue by Trezeguet1** [sora-xor/sora-card-ios#113]
### Preconditions

SORA Card banner appears

### Steps to reproduce

1. Tap on SORA Card banner
2. Enter number
3. Type code
4. Enter email 
5. Verify email
6. Tap on Get XOR button
7. Get XOR by Swap
8. Select assets and enter amount
9. Confirm Swap transaction

### Actual behavior

Swap transaction failed due Get XOR on SORA Card.

![telegram-cloud-photo-size-2-5357372348120026752-y](https://github.com/sora-xor/sora-card-ios/assets/51509139/2f5fe0f5-ea9a-4faa-8767-270af0e589b7)

![telegram-cloud-photo-size-2-5357372348120026761-y](https://github.com/sora-xor/sora-card-ios/assets/51509139/aadda92f-ce99-44d8-9e6a-aa2347cc080a)

![telegram-cloud-photo-size-2-5357372348120026762-y](https://github.com/sora-xor/sora-card-ios/assets/51509139/4ea10286-9012-4070-a40e-f0b191117c5a)

### Expected behavior

Swap transaction should be success due Get XOR on SORA Card.

### Environment

{"label"=>"Development"}

### OS version

16.2

### Additional context

TF Soralution 6(7) version.

**Comment by 2bogd** [comment]
Looks like iban requests are not really related

**Comment by Trezeguet1** [comment]
@2bogd @ShlyapkinIN See which account is signing the transaction.

I transferred 20 vals from this account to a new one through app, then I switched from this account to the created one and switched to SORA card.

Then I made a swap and in the extrinsic the signature of the account from which I switched.

Apparently there is a problem with selecting an account to sign a transaction from the Get XOR dialog in SORA cards.

https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fws.framenode-2.s1.stg1.sora2.soramitsu.co.jp#/explorer
Here I waited and saw the liquidity proxy swap transaction.

![telegram-cloud-photo-size-2-5361589945875156847-y](https://github.com/sora-xor/sora-card-ios/assets/51509139/a9d92150-9045-49c2-aeda-85cf8de33d40)
